### PR TITLE
closes #381 feature straddling checkerboard

### LINKED ISSUE #381: Straddling Checkerboard

### PR DESCRIPTION
**Pull request by t-m-z**

### CODE REVIEW

File: assets/i18n/de.json [hunk 1]
```diff
@@ -2286,6 +2286,24 @@
   "spoonlanguage_description" : "Spielsprache, Geheimsprache",
   "spoonlanguage_example" : "Löffel → lölewöffelewel",
   "spoonlanguage_online_help_url" : "https://blog.gcwizard.net/manual/de/loffelsprache/02-wie-wird-die-loffelsprache-ent-bzw-verschlusselt/",
+  "straddlingcheckerboard_title" : "Straddling Checkerboard",
```
Comment (S-Man42) [inline assets/i18n/de.json:2289]: The German Wikipedia calls this "Spreizung"... What do you mean?

**Comment by S-Man42** [comment]
English lang file is missing.

File: assets/i18n/de.json (hunk 1 above)
Comment (t-m-z) [reply assets/i18n/de.json:2294]: I would say Straddling checkerboard is the correct name in the Community - but you are right, Spreizung will also be found.
I have added Spreizung.

**Comment by S-Man42** [comment]
> 
> 
> English lang file is missing.

Still missing ;)

**Comment by S-Man42** [comment]
Manually merged due to conflicts https://github.com/S-Man42/GCWizard/commit/9324a1535034dccde8b2782df5b32130ec30f9fb

**Pull request closed without merging**